gordo_components/cli/workflow_generator.py

```diff
@@ -182,12 +177,10 @@ def workflow_generator_cli(**kwargs: dict):
 
     # Context requiring pre-processing
     context["target_names"] = [machine.name for machine in config.machines]
+    context["owner_references"] = json.dumps(context["owner_references"])
```

Comment:
Now it is defacto required, so then it should be required in the cli as well

Reply:
If it defaults to `None` then it gets dumped as `null`. Is that not ok?

Reply:
Ahh, tha tjo yeah, I dont know. Inside the workflow there is some tests that owner_reference is set or not, and I dont know what happends if `context["owner_references"]=None`.
```
  {% if owner_references is defined %}
  ownerReferences: {{owner_references}}
  {% endif %}
```
I suspect that it will then set ownerReferences: None, and I dont know what k8s thinks about that.